Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:14.04
+
+ENV POSTGREST_VERSION 0.3.0.4
+ENV POSTGREST_SCHEMA public
+ENV POSTGREST_ANONYMOUS postgres
+ENV POSTGREST_JWT_SECRET thisisnotarealsecret
```

Comment:
The default jwt secret can be overwritten by passing a POSTGREST_JWT_SECRET env variable.